docs/release_process.md

```diff
@@ -7,9 +7,59 @@ This document describes the bpftrace release process.
 We choose to follow semantic versioning. Note that this doesn't matter much for
 major version < 1 but will matter a lot for >= 1.0.0 releases.
 
-See https://semver.org/ .
+See https://semver.org/.
 
-## Branching model
+## Release cadence
+
+bpftrace is released twice a year. Since our biggest dependency, which also
+tends to break things, is LLVM, we align with the [LLVM release
+schedule](https://llvm.org/docs/HowToReleaseLLVM.html). In particular, a
+bpftrace release should happen **two weeks after a major LLVM release**.
+
+In addition, four weeks before the bpftrace release, we create a stabilized
+release branch, which will only receive bug fixes from that moment.
+
+Overview of the release cadence is:
+
+| Task                   | Approximate date                    | Details                                                              |
+| ---------------------- | ----------------------------------- | -------------------------------------------------------------------- |
+| release branch created | **2 weeks before the LLVM release** | [Creating a release branch](#creating-a-release-branch)              |
+| LLVM release           | usually second Tuesday of Mar/Sep   | [LLVM release schedule](https://llvm.org/docs/HowToReleaseLLVM.html) |
+| bpftrace release       | **2 weeks after the LLVM release**  | [Tagging a release](#tagging-a-release)                              |
+
+## Creating a release branch
+
+A release branch should be created four weeks before the planned bpftrace
+release. From that moment, only relevant bug fixes should be backported to the
+branch. Similarly, after the release is done, backport PRs should be opened
+against this branch.
+
+The purpose of the branch is to give sufficient time to test features in the
+upcoming rebase without blocking the development on the master branch.
```

Comment:
"upcoming rebase" - do you mean the new LLVM version?

Reply:
Nope, it's a typo, should be "upcoming release" meaning the new bpftrace release. Will fix.